lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -62,7 +64,8 @@ const NxTextInput = forwardRef<HTMLDivElement, Props>(
             invalid: !isPristine && validatable && isInvalid,
             valid: !isPristine && validatable && !isInvalid,
             disabled: disabled,
-            'nx-text-input--textarea': isTextArea
+            'nx-text-input--textarea': isTextArea,
+            'nx-text-input--date': isDateInput
```

Comment:
Is this class needed?

Reply:
> Is this class needed?

Yup! This was removed when I merged your branch :D